### PR TITLE
Added llms for sentiment and translation

### PR DESCRIPTION
**Pull request by shrav-k**

### CODE REVIEW

File: src/supercontrast/provider/handlers/openai_handler.py [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+# Task.SENTIMENT_ANALYSIS
+
+
+class SentimentAnalysisOutput(pydantic.BaseModel):
+    score: float = pydantic.Field(
+        description="Sentiment score between -1 (very negative) and 1 (very positive)"
+    )
+
+
+SENTIMENT_ANALYSIS_PROMPT = PromptTemplate(
```
Comment (CarterMcClellan) [inline src/supercontrast/provider/handlers/openai_handler.py:45]: I dont think this is how we want to do this.

Llms are bad at modeling continuous variables like a "score"

File: src/supercontrast/provider/handlers/openai_handler.py [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+
+
+class OCROutput(pydantic.BaseModel):
+    text: str = pydantic.Field(description="Extracted text from the image")
+    bounding_boxes: list[dict] = pydantic.Field(
+        description="List of bounding boxes for each detected text"
+    )
+
+
+OCR_PROMPT = PromptTemplate(
```
Comment (CarterMcClellan) [inline src/supercontrast/provider/handlers/openai_handler.py:147]: same logic as before why include a bounding box in this response?

File: src/supercontrast/utils/audio.py [hunk 3]
```diff
@@ -14,3 +18,40 @@ def load_audio_file(audio_file: str) -> str:
...
+    Returns:
+        str: Path to the converted MP3 file.
+    """
+    if output_file is None:
+        output_file = os.path.splitext(input_file)[0] + ".mp3"
+
+    try:
+        subprocess.run(
+            [
+                "ffmpeg",
```
Comment (CarterMcClellan) [inline src/supercontrast/utils/audio.py]: does ffmpeg install have to be part of our wheel now?

Comment (shrav-k) [reply src/supercontrast/utils/audio.py]: this is a utility function I made for personal use, I'll move it to the script

File: src/supercontrast/provider/handlers/openai_handler.py (hunk 2 above)
Comment (shrav-k) [reply src/supercontrast/provider/handlers/openai_handler.py:147]: this is broken, I might just remove it from this PR, I was unable to even get the image to send, just wanted to try this at testing

File: src/supercontrast/provider/handlers/openai_handler.py (hunk 1 above)
Comment (shrav-k) [reply src/supercontrast/provider/handlers/openai_handler.py:45]: Fair enough, I can recommend that they provide a 0 or 1 score

File: src/supercontrast/utils/audio.py (hunk 3 above)
Comment (shrav-k) [reply src/supercontrast/utils/audio.py]: Fixed

File: src/supercontrast/provider/handlers/openai_handler.py (hunk 2 above)
Comment (shrav-k) [reply src/supercontrast/provider/handlers/openai_handler.py:147]: Still doesn't work, but removed bounding boxes. I'll fix this in a future PR

File: src/supercontrast/provider/handlers/openai_handler.py (hunk 1 above)
Comment (shrav-k) [reply src/supercontrast/provider/handlers/openai_handler.py:45]: I fixed this for OpenAI and Anthropic, it now gives a 1, 0, or -1 score

**Pull request merged**